packages/manager/src/views/DashboardPage/DashboardPage.tsx

```diff
@@ -0,0 +1,41 @@
+import {
+  useGetTotalDeployments,
+  useGetDeploymentCounts,
+  useGetDeploymentsTime,
+  useGetMonthlyDeploymentChartWithEphemeral
+} from '@app/services/analytics';
+import { ActivityStream } from './components/ActivityStream';
+import { Analytics } from './components/Analytics';
+
+export const DashboardPage = (): JSX.Element => {
+  const TotalDeploymentData = useGetTotalDeployments();
+  const Totaldeployment = TotalDeploymentData.data?.reduce((acc, obj) => acc + obj.count, 0);
+  const TotalDeploymentCountsData = useGetDeploymentCounts();
+  const TotalProperty = Object.keys(TotalDeploymentCountsData.data || {}).length;
+  const TotalDeploymentsTimeData = useGetDeploymentsTime();
+  const averageTime = TotalDeploymentsTimeData.data?.averageTime;
+  const TotalMonthlyDeploymentData = useGetMonthlyDeploymentChartWithEphemeral().data;
```

Comment:
Approving but before merging. Kindly change this into es6. 

That is, may be like this.
```
 const {data: TotalMonthlyDeploymentData } = useGetMonthlyDeploymentChartWithEphemeral()
```

Reply:
Oopz